src/functions-and-arrays.js

```diff
@@ -1,41 +1,95 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1
+  }
+  else {
+    return number2
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+  if (words.length === 0) { return null }
+  let longestWord = words.sort(function(a, b) { return b.length - a.length; });
```

Comment:
I don't have a problem and actually encourage anybody to go find information online on how to tackle any specific problem.

The only thing I will strongly suggest is for you to understand what the code is doing.

If what `words.sort` is doing is clear to you, then awesome! but it would be a pity if that was not the case.

Remember that we are not testing this LABS, and I do not pretend they are fully completed.

Please if you need any assistance let me know!